lib/workers.js

```diff
@@ -0,0 +1,222 @@
+const { EventEmitter } = require('events');
+const { Worker } = require('worker_threads');
+const Codecept = require('./codecept');
+const Container = require('./container');
+const { getTestRoot } = require('./command/utils');
+const mainConfig = require('./config');
+const output = require('./output');
+const path = require('path');
+const event = require('./event');
+const { Suite, Test } = require('mocha');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [`mocha` import should occur before import of `./codecept`](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4759786)